src/book/03-guides/03-hardware/04-DARwIn-OP2-guide.mdx

```diff
@@ -0,0 +1,328 @@
+  - Upgraded to Intel Atom N2600, faster and efficient
+  - Upgraded to Dynamixel MX Series, higher torque, better durability and control
+
+- Motion and Balance
+
+  - Better Balance
+  - More stable
+
+- Software and AI
+  - Enhanced vision processing software
+  - More accurate ball detection
+
+### DARwIn-OP2 Specifications
+
+- Mainboard: PICO-CV01
+- Cpu: Intel Atom N2600 @1.6GHz single core
+- Ram: Up to 4GB DDR3/240-ping SO-DIMM module (user replaceable)
+- Storage: Half-size mSATA module (32GB) (user replaceable)
+- Sub controller: CM-740
+- Linux mint 17.3 'Rosa' installed on PICO-CV01 board
+
+### Powering on the DARwIn-OP2
+
+To power the DARwIn-OP2, following these steps:
+
+1. Position the DARwIn-OP2 into the ready position.
+   ![The Robotis page showing the ready position of the DARwIn-OP2](./images/darwin_op2_guide/startingPosition.jpg 'Photo from Robotis of the ready position.')
+   <Alert>
+   Make sure you have put the DARwIn into this starting position. Any other position may result in major damage to the robot.
+   </Alert>
+
+2. Connect the 12V DC power supply connector into the DC jack located at the back of the robot.
+   ![The Robotis page showing where the DC connector should go](./images/darwin_op2_guide/dcConnector.jpg 'Photo from Robotis of the location where the DC connector belongs.')
+   <Alert>
+   Make sure the power supply you are using is exactly 12V. Any other power supply may result in major damage to the robot.
+   </Alert>
+
+3. Plug the power supply into the wall socket and flip the switch to on. After this, turn the DARwIn-OP2 power switch on. This is located to the right of the DC jack.
+   ![The Robotis page showing the power switch located on the DARwIn-OP2](./images/darwin_op2_guide/powerSwitch.jpg 'Photo from Robotis of the power switch.')
+
+4. Once the DARwIn-OP2 has booted, the starting position will go stiff and the back panel will light up with different LED colours.
+   ![The Robotis page showing the boot sequence](https://emanual.robotis.com/assets/images/platform/op2/op2_010.png 'Photo from Robotis of the boot sequence.')
+
+### Running the demonstration programs on the DARwIn-OP2
+
+There are different types of operation modes for the DARwIn-OP2.
+
+1. Demonstration-ready mode
+
+   This is the default mode when the DARwIn-OP2 is fully booted and is in the seated position.
+
+   The LEDs on the back panel will light up in the sequence: LED 1 (red), LED 2 (blue), and LED 3 (green).
+   ![The Robotis page showing the demonstration-ready mode LEDs](./images/darwin_op2_guide/demonstrationMode.jpg 'Photo from Robotis of the demonstration-ready mode LEDs.')
+
+2. Autonomous soccer mode
+
+   This mode allows the DARwIn-OP2 the ability to follow and kick a soccer ball. The head camera will follow the ball while the legs move along to follow.
+   While in this mode, the robot may fall. This is not an issue due to the DARwIn-OP2 is programmed to recover by pulling itself from the floor.
```

Comment:
```suggestion
   This mode gives the DARwIn-OP2 the ability to follow and kick a soccer ball. The head camera will follow the ball while the legs move along to follow.
   While in this mode, the robot may fall. This is not an issue because the DARwIn-OP2 is programmed to recover by pulling itself from the floor.
```
just some grammar things, don't forget the batch commit function in GitHub